lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -20,8 +20,12 @@
                                             get_d3_sequential_open_distrib, get_d3_section_grade_distrib,
                                             get_section_display_name, get_array_section_has_problem,
                                             get_students_opened_subsection, get_students_problem_grades,
+                                            get_non_student_list,
                                             )
 from class_dashboard.views import has_instructor_access_for_class
+from courseware.tests.factories import StaffFactory, InstructorFactory
```

Comment:
Can you add these imports to the existing courseware.tests.factories import (a few lines above).